### PR TITLE
some first death message changes

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
first change: if you die to vore, you will not count as the first death
second change: if you die inside something, it will now report your actual coordinates, not just 0,0,0

## Why It's Good For The Game
i don't want to know that john smith died in dorms at 0,0,0

## Changelog
:cl:
add: death to vore won't count as first death
fix: dying inside something wont report it as 0,0,0
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/blackbox.dm [hunk 1]
```diff
@@ -294,12 +294,12 @@ Versioning
 		return
 	if(!L || !L.key || !L.mind)
 		return
-	if(!L.suiciding && !first_death.len)
+	if(!L.suiciding && !first_death.len && (!istype(L.loc, /obj/belly))
```
Comment (Linzolle) [inline code/controllers/subsystem/blackbox.dm]: missing a parenthesis here 
```suggestion
	if(!L.suiciding && !first_death.len && (!istype(L.loc, /obj/belly)))
```

File: code/controllers/subsystem/blackbox.dm [hunk 2]
```diff
@@ -294,12 +294,12 @@ Versioning
...
 	if(!L || !L.key || !L.mind)
 		return
-	if(!L.suiciding && !first_death.len)
+	if(!L.suiciding && !first_death.len && (!istype(L.loc, /obj/belly)))
 		first_death["name"] = "[(L.real_name == L.name) ? L.real_name : "[L.real_name] as [L.name]"]"
 		first_death["role"] = null
 		if(L.mind.assigned_role)
 			first_death["role"] = L.mind.assigned_role
-		first_death["area"] = "[AREACOORD(L)]"
+		first_death["area"] = "[AREACOORD(get_turf(L))]"
```
Comment (DeltaFire15) [inline code/controllers/subsystem/blackbox.dm:302]: CI:
code/controllers/subsystem/blackbox.dm:302:error: bad embedded expression [get_turf(L).]

**Comment by DeltaFire15** [comment]
Marking PR as stale due to CI-failing state combined with a long period of no commits.

**Comment by DeltaFire15** [comment]
Closing this PR due to staleness.

**Pull request closed without merging**